docroot/themes/custom/twentyeighteen/sass/components/call-to-action/_call-to-action.scss

```diff
@@ -0,0 +1,34 @@
+// Call to Action Block
+//
+// Styling for three call to action blocks on bottom of homepage
+//
+// Markup: call-to-action.twig
+//
+// Style guide: components.call_to_action
+
+.call-to-action-block {
+  @include span(4 of 12);
+
+  /*  position: relative; */
+}
+
+.call-to-action-text {
+  /* position: absolute; */
+  margin-top: -30%;
```

Comment:
not 100% sure how I feel about this using margins vs absolute positioning. I'm wondering if we can arrange the markup to make this easier. (see below comment on suggested markup)